lib/active_record/tasks/sqlserver_database_tasks.rb

```diff
@@ -49,9 +49,11 @@ def purge
       end
 
       def structure_dump(filename, extra_flags)
+        server_arg = "-S #{Shellwords.escape(configuration['host'])}"
+        server_arg += ":#{Shellwords.escape(configuration['port'])}" if configuration['port']
```

Comment:
Should the shovel operator `<<` be used here instead?

Reply:
When strings are frozen you cannot use `<<` because it will attempt to modify the string, whereas `+=` returns a new string.

Reply:
From what I see here, it's just an interpolated string, where does the freeze come from? I apologize if I'm missing something.

Reply:
Frozen strings will also be the default in a future version of Ruby, so it's just good practice to behave as if that is already true to future-proof the code a bit.